pkg/Microsoft.Private.CoreFx.OOB/Microsoft.Private.CoreFx.OOB.pkgproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <!-- We use the PackageVersion property defined in Packaging.props for the Prerelease packages which are using it to be built -->
+    <_PreReleasePackageVersion>$(PackageVersion)</_PreReleasePackageVersion>
+    <!-- We don't need to harvest the stable packages to build this -->
+    <HarvestStablePackage>false</HarvestStablePackage>
+  </PropertyGroup>
+  <ItemDefinitionGroup>
+    <PrereleaseLibraryPackage>
+      <Version>$(_PreReleasePackageVersion)</Version>
+    </PrereleaseLibraryPackage>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <PkgProjGlobPath Include="$(SourceDir)/**/*.pkgproj" />
```

Comment:
This is not necesarily right though, we build packages for some things that are not technically OOB like Collections.Immutable and ReflectionMetadata